storage/src/store/mod.rs

```diff
@@ -0,0 +1,79 @@
+//! A simple append-only key-value store backed by a log of state-change operations.
+//!
+//! This store provides append-only functionality without requiring MMR (Merkle Mountain Range) for
```

Comment:
nit: I prefer describing `store` as-is (without any relation to MMRs) and then adding a recommendation to use `adb` if you want to authenticate data (the same could be done in reverse in the `adb/mod.rs` for "just need a store (without overhead of maintaining proofs)?"

https://github.com/commonwarexyz/monorepo/blob/31c6abdf1a264beb727817cdd20b34295b3a3fe4/storage/src/journal/fixed.rs#L7